controllers/updates/views/listing/press-releases.njk

```diff
@@ -105,10 +105,10 @@
 
                     {% set pressCopy = __('global.contact.press') %}
                     {% call card(pressCopy.title) %}
-                        <div class="s-prose">
+                        <div class="s-prose u-text-small">
                             {% for region in pressCopy.regions %}
-                                <p><strong>{{ region.title }}</strong></p>
-                                <dl class="o-definition-list o-definition-list--wrapped u-margin-bottom">
+                                <h4 class="t5">{{ region.title }}</h4>
+                                <dl class="o-definition-list o-definition-list--wrapped">
```

Comment:
I think this still needs the `u-margin-bottom` class - they look a bit tight without it, eg:

![image](https://user-images.githubusercontent.com/394376/49949976-ce343680-feee-11e8-9c40-9d550c95d292.png)

Reply:
Here's the original design, for reference:

![image](https://user-images.githubusercontent.com/394376/49950044-f6239a00-feee-11e8-8ef5-a7da92c24874.png)